content/en-us/production/publishing/vr-guidelines.md

```diff
@@ -48,7 +48,7 @@ The following best practices may help you reach 72 frames per second with a high
 
 - Enable [instance streaming](../../workspace/streaming.md) so that content is streamed in and out dynamically on the device.
 - Be careful to not overuse CPU-heavy computations like [raycasting](../../workspace/raycasting.md).
-- Avoid the usage of `Library.task.wait()` over `Class.RunService` frame updates. `Library.task.wait()` will not give you precise results compared to connecting to `Class.RunService.Stepped`.
+- Avoid the usage of `Library.task.wait()` over `Class.RunService` frame updates. `Library.task.wait()` will not give you precise results compared to connecting to `Class.RunService.PreSimulation`.
```

Comment:
`task.wait()` is equivalent to `RunService.Heartbeat:Wait()` so I am not sure I fully understand this sentence

Reply:
Was just updating what was already there. Maybe we could rework/remove this sentence given that task.wait() is just as effective as the RunService events now.